Image `saturate` / `desaturate` appear on the doc, but not on the module...

# Expected Behavior
Hi, I would like to reduce or increase my image saturation, as specified in the doc [here](https://github.com/oliver-moran/jimp/tree/master/packages/jimp#advanced-usage), but the module doesn't have those options.  
My issue refer to another one, previously closed: https://github.com/oliver-moran/jimp/issues/968

# Current Behavior
`Type '"saturate"' is not assignable to type 'ColorActionName'.` because `saturate` or `desaturate` (and `lighten`, `darken`) doesn't exist on `ColorActionName`.

# Failure Information (for bugs)

\-\-\-

## Steps to Reproduce
Try to change the saturation of an image
```js
image.color([{ apply: 'saturate', params: [10] }]);
```
As you can see on the screen, even Intellisense doesn't propose me `saturate`, `desaturate`...
**Screenshots**  
![Intellisense](https://user-images.githubusercontent.com/29955402/107883637-5445d080-6ef0-11eb-8c52-06a0e40feb12.png)

## Context
- Jimp Version: 0.16.1
- Operating System: macOS Big Sur 11.2
- Node version: v14.15.1

## Failure Logs
```
$ npm start
>>> src/helpers/FilterManager.ts(95,17): error TS2322: Type '"saturate"' is not assignable to type 'ColorActionName'.
```

--- a/packages/jimp/types/test.ts
+++ b/packages/jimp/types/test.ts
@@ -1,4 +1,5 @@
 import * as Jimp from "jimp";
+import { ColorActionName } from "@jimp/plugin-color";
 
 const jimpInst: Jimp = new Jimp("test");
 
@@ -11,6 +12,7 @@ jimpInst.displace(jimpInst, 2);
 jimpInst.shadow((err, val, coords) => {});
 jimpInst.fishEye({ r: 12 });
 jimpInst.circle({ radius: 12, x: 12, y: 12 });
+jimpInst.color([{ apply: ColorActionName.SATURATE, params: [90] }]);
 // $ExpectError
 jimpInst.PNG_FILTER_NONE;
 

--- a/packages/plugin-color/index.d.ts
+++ b/packages/plugin-color/index.d.ts
@@ -1,16 +1,22 @@
 import { ImageCallback } from "@jimp/core";
 
-type ColorActionName =
-  | "mix"
-  | "tint"
-  | "shade"
-  | "xor"
-  | "red"
-  | "green"
-  | "blue"
-  | "hue"
-  | "lighten"
-  | "darken";
+export enum ColorActionName {
+  LIGHTEN = "lighten",
+  BRIGHTEN = "brighten",
+  DARKEN = "darken",
+  DESATURATE = "desaturate",
+  SATURATE = "saturate",
+  GREYSCALE = "greyscale",
+  SPIN = "spin",
+  HUE = "hue",
+  MIX = "mix",
+  TINT = "tint",
+  SHADE = "shade",
+  XOR = "xor",
+  RED = "red",
+  GREEN = "green",
+  BLUE = "blue",
+}
 
 type ColorAction = {
   apply: ColorActionName;

--- a/packages/plugin-color/src/index.js
+++ b/packages/plugin-color/src/index.js
@@ -128,6 +128,24 @@ function colorFn(actions, cb) {
   return this;
 }
 
+export const ColorActionName = Object.freeze({
+  LIGHTEN: "lighten",
+  BRIGHTEN: "brighten",
+  DARKEN: "darken",
+  DESATURATE: "desaturate",
+  SATURATE: "saturate",
+  GREYSCALE: "greyscale",
+  SPIN: "spin",
+  HUE: "hue",
+  MIX: "mix",
+  TINT: "tint",
+  SHADE: "shade",
+  XOR: "xor",
+  RED: "red",
+  GREEN: "green",
+  BLUE: "blue",
+})
+
 export default () => ({
   /**
    * Adjusts the brightness of the image

--- a/packages/plugin-color/src/index.js
+++ b/packages/plugin-color/src/index.js
@@ -144,7 +144,7 @@ export const ColorActionName = Object.freeze({
   RED: "red",
   GREEN: "green",
   BLUE: "blue",
-})
+});
 
 export default () => ({
   /**

